lesson7/task2/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lesson 7, task 2</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <main class="main">
+        <div class="main">
+            <div class="title">
+                <p class="title">LOGIN FORM</p>
+            </div>
+            <div class="login">
+                <div class="buttons">
+                    <button class="fb"><img src="images/fb-icon.png" alt="">Login with Facebook</button>
+                    <button class="tw"><img src="images/twitter-icon.png" alt="">Login with Twitter</button>
+                </div>
+            </div>
+            <div class="or"><span class="or">Or</span></div>
+            <div class="account">
+                <input type="text" name="username" id="username" value="username">
+                <input type="text" name="password" id="password" value="password">
```

Comment:
There should be type "password". And both of them should have "placeholder" instead of "value"